src/mesh/smoothing.rs

```diff
@@ -826,12 +830,13 @@ mod tests {
             Vector3::new(2.0, 2.0, 2.0),
             2,
             3,
+            NormalStrategy::Sharp,
         );
         let v2v = topology::compute_vertex_to_vertex_topology(&mesh);
         let v2f = topology::compute_vertex_to_face_topology(&mesh);
         let f2f = topology::compute_face_to_face_topology(&mesh, &v2f);
 
-        let subdivided_mesh = loop_subdivision(&mesh, &v2v, &f2f)
+        let subdivided_mesh = loop_subdivision(&mesh, &v2v, &f2f, NormalStrategy::Smooth)
```

Comment:
The reason I asked you to parametrize `laplacian_smoothing` and `loop_subdivision` with normal strategy is so that we don't have to modify the tests and snapshots by still using the `NormalStrategy::Sharp` here.